docs/skopeo.1.md

```diff
@@ -29,10 +33,20 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   By default, uses the authorization state in `$HOME/.docker/config.json`,
   which is set e.g. using `(docker login)`.
 
+  **docker-archive:**_docker-reference_
+  An image _tag_ stored in the local docker archives
+
+  **docker-daemon:**_docker-reference_
+  An image _tag_ stored in the docker daemon internal storage
+
   **oci:**_path_**:**_tag_
   An image _tag_ in a directory compliant with "Open Container Image
   Layout Specification" at _path_.
 
+  **ostree:**_path_
+  An image in local ostree format.  Default location is
```

Comment:
An image in a local OSTree repository.

The format is `**ostree:**_image_` or `**ostree:**_image_**@/absolute/repo/path`, with `/absolute/repo/path` defaulting to `/ostree/repo`. _image_ is not quite a dockerlike-reference (e.g. no `docker.io/library` normalization is used), but it either should contain a `:`_tag_, or an implied `:latest` is used.